Client/Frontend/Theme/Theme.swift

```diff
@@ -236,8 +236,23 @@ class HomePanelColor {
     var toolbarTint: UIColor { return UIColor.Grey50 }
 
     var panelBackground: UIColor { return UIColor.White }
+    var separatorColor: UIColor {
```

Comment:
It is not clear what the difference between separatorColor and separator. Could you please rename it to have more understandable titles.

Reply:
`separator` is somehow broken. It returns completely different value that what the code indicates. I don't know why.

Reply:
Maybe this is iOS 13 issue. Do you see the same behavior in iOS 12 also?